hail/python/hailtop/batch_client/aioclient.py

```diff
@@ -608,7 +611,8 @@ async def list_batches(self, q=None):
             body = await resp.json()
 
             for batch in body['batches']:
-                yield Batch(self, batch['id'], attributes=batch.get('attributes'), n_jobs=int(batch['n_jobs']))
+                yield Batch(self, batch['id'], attributes=batch.get('attributes'),
```

Comment:
I'm not sure if `batch` here is the same as the `status` we actually want cached.

Reply:
From my reading of it, they are.

The result of `hailctl curl default batch /api/v1alpha/batches | jq '.batches[0]'` and `hailctl curl default batch /api/v1alpha/batches/$ID` are the same (`$ID` is the id of the first batch returned by the `batches` endpoint).

The backend uses the same select and join expressions on the code paths for these endpoints. Furthermore, they are both processed through `batch_record_to_dict`.